packages/rum-recorder/src/domain/rrweb/observer.ts

```diff
@@ -39,6 +39,8 @@ import {
   throttle,
 } from './utils'
 
+// TODO: remove this global MutationBuffer instance
+// eslint-disable-next-line local-rules/enforce-declarative-modules
```

Comment:
I want to remove this in a separate PR, because it involves a bit of rearchitecturing of RRWeb codebase